modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -3,25 +3,26 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-package com.liferay.source.formatter.dependencies.upgrade;
-
 import com.liferay.commerce.constants.CommerceOrderPaymentConstants;
 
-import com.liferay.portal.kernel.util.PortalUtil;
-
 import com.liferay.document.library.kernel.model.DLVersionNumberIncrease;
-import java.util.Date;
+
+import com.liferay.document.library.util.DLURLHelperUtil;
+
+import com.liferay.portal.kernel.search.Sort;
 
 import com.liferay.portal.kernel.service.ServiceContextThreadLocal;
 
 import com.liferay.portal.kernel.util.HtmlParser;
 
-import com.liferay.portal.kernel.search.Sort;
+import com.liferay.portal.kernel.util.PortalUtil;
 
-import com.liferay.document.library.util.DLURLHelperUtil;
+import java.util.Date;
 
 import org.osgi.service.component.annotations.Reference;
 
+package com.liferay.source.formatter.dependencies.upgrade;
```

Comment:
@MicaelleSilvaa, the `package` shouldn't be included in the sort only the `imports` :wink: 

Additionally, please start the commit messages with capital letter, e.g. `Test case ...`

Reply:
Resolved 😉